src/gurt/misc.c

```diff
@@ -81,6 +81,25 @@ d_asprintf(char **strp, const char *fmt, ...)
 	return rc;
 }
 
+char *
+d_asprintf2(int *_rc, const char *fmt, ...)
```

Comment:
Should the old `d_asprintf` be removed?

Reply:
I wasn't sure if there was any middleware using it.

Reply:
Right, that's public. Then we shouldn't remove. If anything, maybe the old should be a wrapper for the new? But maybe overkill